data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -72,7 +76,13 @@ podman run -d --net host --privileged --name httpd \
 # Set CACHEURL to the default route, so we try to consume any images cached on the host
 # running the VM (dev-scripts configures a cache here), if none is found then the
 # downloader containers just skip and download from the internet location
-CACHEURL="http://$(ip r | grep $PROVISIONING_NIC | awk '/default/ {print $3};')/images"
+IPTABLES=iptables
+IP=$(ip r | grep $PROVISIONING_NIC | awk '/default/ {print $3};')
+if ! ping -c 1 $IP ; then
+    IP="[$(ip -6 r | grep $PROVISIONING_NIC | awk '/default/ {print $3};')]"
```

Comment:
Just a comment, this doesn't work for me, as i don't seem to have any ipv6 default route (even if i enabled router advertisements). However, it worked with an alternative command: /sbin/ip -6 addr show provisioning | grep inet6 | awk -F '[ \t]+|/' '{print $3}' | grep -v ^::1 | grep -v ^fe80

Reply:
thanks, will update